fenix/app/src/main/java/org/mozilla/fenix/customtabs/ExternalAppBrowserFragment.kt

```diff
@@ -66,7 +66,7 @@ class ExternalAppBrowserFragment : BaseBrowserFragment() {
             feature = CustomTabsIntegration(
                 store = requireComponents.core.store,
                 useCases = requireComponents.useCases.customTabsUseCases,
-                toolbar = browserToolbar,
+                toolbar = browserToolbarView.view,
```

Comment:
🔍 nit: unrelated to this review.  Since `BrowserToolbar` is now in `BrowserToolbarView`.  Maybe we rename `browserToolbarView.view` to `browserToolbarView.toolbar`.

Reply:
filled a [follow up](https://bugzilla.mozilla.org/show_bug.cgi?id=1884455)